semanticcpg/src/main/scala/io/shiftleft/semanticcpg/passes/languagespecific/fuzzyc/MethodStubCreator.scala

```diff
@@ -3,44 +3,48 @@ package io.shiftleft.semanticcpg.passes.languagespecific.fuzzyc
 import io.shiftleft.codepropertygraph.Cpg
 import io.shiftleft.codepropertygraph.generated.nodes.{NewBlock, NewMethodReturn}
 import io.shiftleft.codepropertygraph.generated.{EdgeTypes, EvaluationStrategies, NodeTypes, nodes}
-import io.shiftleft.passes.{CpgPass, DiffGraph}
+import io.shiftleft.passes.{CpgPass, DiffGraph, ParallelIteratorExecutor}
 import org.apache.logging.log4j.{LogManager, Logger}
 import org.apache.tinkerpop.gremlin.structure.Direction
 import io.shiftleft.semanticcpg.language._
 
+import scala.collection.concurrent.TrieMap
 import scala.jdk.CollectionConverters._
+import scala.collection.parallel.CollectionConverters._
 
 /**
   * This pass has no other pass as prerequisite.
   */
 class MethodStubCreator(cpg: Cpg) extends CpgPass(cpg) {
-  import MethodStubCreator.logger
 
   private case class NameAndSignature(name: String, signature: String)
 
   // Since the method fullNames for fuzzyc are not unique, we do not have
   // a 1to1 relation and may overwrite some values. We deem this ok for now.
-  private var methodFullNameToNode = Map[String, nodes.MethodBase]()
-  private var methodToParameterCount = Map[NameAndSignature, Int]()
+  private var methodFullNameToNode = scala.collection.concurrent.TrieMap[String, nodes.MethodBase]()
```

Comment:
```suggestion
  private var methodFullNameToNode = TrieMap.empty[String, nodes.MethodBase]
```

Reply:
This doesn't look right: `methodToParameterCount` disappears.